biz.aQute.launcher/src/aQute/launcher/constants/LauncherConstants.java

```diff
@@ -44,11 +44,12 @@
 	final static String				LAUNCH_NAME					= "launch.name";
 	final static String				LAUNCH_NOREFERENCES			= "launch.noreferences";
 	final static String				LAUNCH_NOTIFICATION_PORT	= "launch.notificationPort";
+	final static String				LAUNCH_EAGER_ACTIVATION		= "launch.eager.activation";
```

Comment:
Would `launch.activation.eager` make more sense? General to specific.

```suggestion
	final static String				LAUNCH_EAGER_ACTIVATION		= "launch.activation.eager";
```

Reply:
I was used the exact property name specified by @bjhargrave!?!

Reply:
Ah. My suggestion is better though!

Reply:
:) but I'm open for naming options, as long as this passes review before release ;)

Reply:
I like Sean's naming suggestion.